### PR TITLE
Adding sample of pie chart, 3D pie chart, bar of pie chart

### PR DESCRIPTION
**Pull request by sharonaumao**
Added KB documentation samples. The samples are:
- Create-pie-chart
- Create-3D-pie-chart
- Create-bar-of-pie-chart

### CODE REVIEW

File: Charts/Create-3D-pie-chart/.NET-Standard/Create-3D-pie-chart/Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                chart.ChartData.SetValue(6, 2, 28);
+                chart.ChartData.SetValue(7, 2, 9);
+                //Set region of Chart data.
+                chart.DataRange = chart.ChartData[2, 1, 7, 2];
+                //Set chart series in the column for assigned data region.
+                chart.IsSeriesInRows = false;
+                //Set a Chart Title.
+                chart.ChartTitle = "3D Pie Chart";
+                //Set Datalabels.
+                IOfficeChartSerie series1 = chart.Series[0];
```
Comment (Suriya-Balamurugan) [inline Charts/Create-3D-pie-chart/.NET-Standard/Create-3D-pie-chart/Program.cs:43]: If only one series means, don't use the variable name as 'series1'

Use variable name as 'serie'

File: Charts/Create-bar-of-pie-chart/.NET-Standard/Create-bar-of-pie-chart/Program.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                chart.ChartData.SetValue(6, 2, 28);
+                chart.ChartData.SetValue(7, 2, 9);
+                //Set region of Chart data.
+                chart.DataRange = chart.ChartData[2, 1, 7, 2];
+                //Set chart series in the column for assigned data region.
+                chart.IsSeriesInRows = false;
+                //Set a Chart Title.
+                chart.ChartTitle = "Bar of Pie Chart";
+                //Set Datalabels.
+                IOfficeChartSerie series1 = chart.Series[0];
```
Comment (Suriya-Balamurugan) [inline Charts/Create-bar-of-pie-chart/.NET-Standard/Create-bar-of-pie-chart/Program.cs:43]: If only one series means, don't use the variable name as 'series1'

Use variable name as 'serie'

File: Charts/Create-pie-chart/.NET-Standard/Create-pie-chart/Program.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                chart.ChartData.SetValue(6, 2, 28);
+                chart.ChartData.SetValue(7, 2, 9);
+                //Set region of Chart data.
+                chart.DataRange = chart.ChartData[2, 1, 7, 2];
+                //Set chart series in the column for assigned data region.
+                chart.IsSeriesInRows = false;
+                //Set a Chart Title.
+                chart.ChartTitle = "Pie Chart";
+                //Set Datalabels.
+                IOfficeChartSerie series1 = chart.Series[0];
```
Comment (Suriya-Balamurugan) [inline Charts/Create-pie-chart/.NET-Standard/Create-pie-chart/Program.cs:43]: If only one series means, don't use the variable name as 'series1'

Use variable name as 'serie'

File: Charts/Create-3D-pie-chart/.NET-Standard/Create-3D-pie-chart/Program.cs (hunk 1 above)
Comment (sharonaumao) [reply Charts/Create-3D-pie-chart/.NET-Standard/Create-3D-pie-chart/Program.cs]: @Suriya-Balamurugan - the change is addressed

File: Charts/Create-bar-of-pie-chart/.NET-Standard/Create-bar-of-pie-chart/Program.cs (hunk 2 above)
Comment (sharonaumao) [reply Charts/Create-bar-of-pie-chart/.NET-Standard/Create-bar-of-pie-chart/Program.cs]: @Suriya-Balamurugan - the change is addressed

File: Charts/Create-pie-chart/.NET-Standard/Create-pie-chart/Program.cs (hunk 3 above)
Comment (sharonaumao) [reply Charts/Create-pie-chart/.NET-Standard/Create-pie-chart/Program.cs]: @Suriya-Balamurugan -the change is addressed

**Comment by Suriya-Balamurugan** [comment]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**